src/adapters/pagarme/risk-analysis.js

```diff
@@ -3,18 +3,24 @@ const {
   always,
   applySpec,
   assoc,
-  ifElse,
-  propEq,
-  pipe,
+  cond,
   of,
+  pathEq,
+  pipe,
+  prop,
+  T,
 } = require('ramda')
 
-const isIndividual = propEq('document_type', 'cpf')
+const isIndividual = pathEq(['recipient', 'document_type'], 'cpf')
 
-const policyId = ifElse(isIndividual, always(6), always(5))
+const getPolicyId = cond([
+  [prop('policyId'), prop('policyId')],
+  [isIndividual, always(6)],
+  [T, always(5)],
+])
```

Comment:
Do we really need to use Ramda for this?

Reply:
Why not

```js
const getPolicyId = riskAnalysis => {
  if (riskAnalysis. policyId) {
    return riskAnalysis. policyId
  }

  if (isIndividual(riskAnalysis)) {
    return 6
  }

  return 5
}
```

Reply:
> Do we really need to use Ramda for this?

Yes ✨

Reply:
Both ways are great, but the file already uses Ramda to build the spec, shouldn't we follow the pattern?

Reply:
No

Reply:
Ramda's the language now. Ramda is beautiful language :heart: I'd get worried if people started using javascript.